src/bun.js/webcore/request.zig

```diff
@@ -77,6 +77,33 @@ pub const Request = struct {
     pub const getBlob = RequestMixin.getBlob;
     pub const getFormData = RequestMixin.getFormData;
     pub const getBlobWithoutCallFrame = RequestMixin.getBlobWithoutCallFrame;
+    pub const WeakRef = struct {
```

Comment:
This should probably be called Optional or something like that 

And then it would still increment the reference count

Reply:
If we do that would only be a strong ref not a weak ref, and would keep more memory alive without needing.